mobile/src/navigations/root/root.tsx

```
@@ -16,7 +20,40 @@ const screenOptions: NativeStackNavigationOptions = {
   headerShown: false,
 };
 
-const Root: React.FC = () => {
+type RootProperties = {
+  routeName: string | undefined;
+};
+
+const Root: React.FC<RootProperties> = ({ routeName }) => {
+  const dispatch = useAppDispatch();
+  const { authenticatedUserDataStatus } = useAppSelector(({ auth }) => ({
+    authenticatedUserDataStatus: auth.authenticatedUserDataStatus,
+  }));
+
+  const isRoot = routeName === RootScreenName.SIGN_IN;
+
+  useEffect(() => {
+    if (isRoot) {
+      void dispatch(userActions.loadAll()).catch((error: Error) => {
+        throw new Error(`Error loading all users: ${error.message}`);
+      });
```

I don't think we should throw an error here